packages/components/src/Button/Button.spec.js

```diff
@@ -0,0 +1,27 @@
+import { render } from '@testing-library/react';
+import { ThemeProvider, createTheme } from '@platzily-ui/styling';
+import Button from './Button';
+
+describe('@components/Button', () => {
+  const spy = {};
+
+  beforeEach(() => {
+    spy.console = jest.spyOn(console, 'error').mockImplementation(() => {});
+  });
+
+  afterEach(() => {
+    spy.console.mockRestore();
+  });
+
+  it('Given a normal call it should render properly', () => {
+    // act
+    render(
+      <ThemeProvider theme={createTheme()}>
+        <Button type="button" />
+      </ThemeProvider>,
+    );
+
+    // assert
+    expect(spy.console).not.toBeCalled();
```

Comment:
Why are we testing console.error not be called? The component never call that method. We could test the button functionality better, like for example, if the user click the button the onClick prop should be called, if the button is disabled it should not be called.

Reply:
Besides to what @omarefg said, let's apply BDD for our tests, because the description of the tests should highlight the use cases of each component, so let's use the framework that suggests using the `given-when-then` approach to create the descriptions. Let's remember that should put our focus on creating functional tests.

ref: [BDD and functional testing](https://medium.com/javascript-scene/behavior-driven-development-bdd-and-functional-testing-62084ad7f1f2)